managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerErrorsTest.java

```diff
@@ -329,40 +329,6 @@ public void addComplete(Position position, Object ctx) {
         latch.await();
     }
 
-    @Test
-    public void recoverAfterWriteError() throws Exception {
```

Comment:
test not  relevant anymore since we are not going to automatically fail entries for a fix amount of time after a failure

Reply:
Sure, I was meaning to convert this test to use the `readyToCreateNewLedger()`, so that we cover the basic (mocked) test here

Reply:
@merlimat i have already modified the following test to use readToCreateNewLedger().  Take a look at:
https://github.com/apache/pulsar/pull/5322/files#diff-6e96907493eb92d48f2f448d6bbd3b44L413

Reply:
Got it .